src/gc.c

```diff
@@ -2723,8 +2753,12 @@ mark: {
             }
 #ifdef COPY_STACKS
             void *stkbuf = ta->stkbuf;
-            if (stkbuf && ta->copy_stack)
+            if (stkbuf && ta->copy_stack) {
                 gc_setmark_buf_(ptls, stkbuf, bits, ta->bufsz);
+                // TODO: attribute size of stack
+                // TODO: edge to stack data
+                // TODO: synthetic node for stack data (how big is it?)
```

Comment:
Can you add a mention that these TODO are specifically related only to `gc_heap_snapshot_record`?